File: src/main/csharp/HU.OOP.Week1.Opdracht5/Student.cs
namespace HU.OOP.Week1.Opdracht4;

public class Student
{
    public String Naam { get; private set; } 
    public int Nummer { get; set; }

    public Student(String nm)
    {
        this.Naam = nm;
    }

    public Student(String nm, int nr) {
        Naam = nm;
        Nummer = nr;
    }

    public override string ToString()
    {
        return "Deze student heet " + Naam + " en heeft nummer " + Nummer;
    }
}

File: src/main/java/nl/hu/oop/werkboek/week1/les1/opdracht5/Student.java
package nl.hu.oop.werkboek.week1.les1.opdracht5;

public class Student {

    private String naam;
    private int studentnr;

    public Student(String nm) {
        naam = nm;
    }

    public Student(String nm, int nr) {
        naam = nm;
        studentnr = nr;
    }

    public String getNaam() {
        return naam;
    }

    public String toString() {
        return "Deze student heet " + naam + " en heeft nummer " + this.studentnr;
    }

    public int getStudentNummer() {
        return studentnr;
    }

    public void setStudentNummer(int nr) {
        studentnr = nr;
    }
}
